conf/bookkeeper.conf

```diff
@@ -224,7 +224,7 @@ maxPendingAddRequestsPerThread=10000
 auditorPeriodicBookieCheckInterval=86400
 
 # The number of entries that a replication will rereplicate in parallel.
-rereplicationEntryBatchSize=5000
+rereplicationEntryBatchSize=1000
```

Comment:
I would suggest reducing this to 100. because this setting is applied per ledger.

Reply:
fixed